rudder-webapp/SPECS/rudder-webapp.spec

```diff
@@ -436,6 +436,18 @@ if [ $1 -eq 0 ]; then
     groupdel %{config_repository_group}
     echo " Done"
   fi
+
+%if 0%{?sles_version}
+  # Remove required includes in the SLES apache2 configuration
+  if grep -qE "^. /etc/sysconfig/rudder-apache$" /etc/sysconfig/apache2; then
```

Comment:
The grep is already done by the sed line since sed won't do anything if there is no change.
Shouldn't we check the existence of the file instead ? (in case the user remove apache at the same time)